src/components/Label/Label.tsx

```diff
@@ -2,25 +2,30 @@ import React from "react";
 
 import classNames from "classnames";
 
-export interface LabelProps {
+export type LabelProps = {
```

Comment:
is there an advantage for doing types over interfaces? the linter suggests using interfaces over types.

Reply:
My two cents on this. It's largely just a heuristic based on personal preference. See: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces
I typically only go with `interface` when I know I'll need to extend (even though you really can do that with both).
Other than that I prefer `type` because it's less typing (heh).

Reply:
Ok so read that and some other opinions/guidance out there and yeah seems like a personal preference. Interfaces can't describe primitives so you have to use types primitives. Apparently the only  thing types can't really do is declaration merging which sounds like mostly dangerous language magic if you ask me.. Types to me feel slightly more familiar and are slightly more concise but that's just personal preference. Happy to prefer either.

Reply:
I think I'm more old school in thought. I know that now `type` can be extended like an `interface` now, but it seems wrong to extend something that's not an interface. I've been using interfaces to define basic objects (e.g. the classic Point example); and types for literals or intersections when it doesn't make sense to extend. 

I just read a recommendation that said to use interfaces for public API use so that consumers can extend them (although that's maybe moot now) and types for props/state. I can live with `type`ing everything though if we decide to move forward with that. My only change request is to update the lint rules so it doesn't underline this anymore if you want to use `type`.

Reply:
_slack discussion occurs_ -> we'll prefer interfaces for objects and types for literals or intersections when it doesn't make sense to extend.

Reply:
Alright, updated Label and eslintrc